src/registrar/forms/application_wizard.py

```diff
@@ -190,7 +190,7 @@ def clean(self):
                     "You can’t complete this application yet. "
                     "Only tribes recognized by the U.S. federal government "
                     "or by a U.S. state government are eligible for .gov "
-                    'domains. Please use our <a href="{}">contact form</a> to '
+                    'domains. Please use our <a href="{% public_site_url 'contact' %}">contact form</a> to '
```

Comment:
This markup is only valid in html files. For python strings, you'd want to use something like `f"domains. Please use our <a href="{your_function_here()}">contact form</a> to "` and that should be semantically equivalent. I think, if the @register.simple_tag doesn't break it, you should be able to call the public_site_url directly like my example but if not I'd be glad to assist.

Reply:
Thank you! Rachid helped make this update and I think he's going to tag you on that PR.